libs/remote_storage/src/s3_bucket.rs

```diff
@@ -709,11 +709,15 @@ impl RemoteStorage for S3Bucket {
         let body = StreamBody::new(from.map(|x| x.map(Frame::data)));
         let bytes_stream = ByteStream::new(SdkBody::from_body_1_x(body));
 
+        let key = self.relative_path_to_s3_object(to);
+
+        tracing::info!("Uploading {from_size_bytes} bytes to absolute {key}");
```

Comment:
nit: what does "absolute" refer to here?

Reply:
This was a debug logging commit, dropped now.

(absolute meant including the prefix_in_bucket)